assets/stylesheets/components/evolution/nav_drop_carousel/_nav_drop_carousel.scss

```diff
@@ -0,0 +1,370 @@
+
+
+@import 'nav_drop_carousel_config';
```

Comment:
@OscarRobertRodriguez 
Naming is like this: `+@import 'nav_drop_carousel-config'`, notice a dash before the 'config' word.
Also, the file that is being imported should be renamed accordindly.